BaseApi/EventSubscriber/AbstractSubscriber.php

```diff
@@ -59,6 +48,6 @@ protected function extractAnnotation(KernelEvent $event, $annotationClassName)
      */
     protected function eventElligible(KernelEvent $event)
     {
-        return $event->isMasterRequest() && $this->isApiRequest($event->getRequest());
```

Comment:
Yes, no more check.

As we use the cacheReader for the annotations, the impact is less important.

The main reason for this change is to allow every integartor to create it's own api without taking care of the route name